packages/core/src/components/dropdown/dropdown.scss

```
@@ -32,8 +32,10 @@
   }
 }
 
+$dropdown-offset: 3.125rem;
```

This solution seems fine after all.
Can we change the offset to 3rem to be more inline with other spacings?
This would require updating the visual test snapshots.

```suggestion
$dropdown-offset: 3rem;
```

---

Hi, yes I will update it to 3, test and update in the PR